ROMFS/px4fmu_common/init.d-posix/airframes/4008_gz_advanced_plane

```diff
@@ -16,47 +16,43 @@ param set-default SENS_EN_MAGSIM 1
 param set-default SENS_EN_ARSPDSIM 1
 
 param set-default FW_LND_ANG 8
-param set-default NPFG_PERIOD 12
 
-param set-default FW_MAN_P_MAX 30
-param set-default FW_PR_P 0.9
-param set-default FW_PR_FF 0.5
-param set-default FW_PR_I 0.5
+param set-default FW_PR_FF 0.08
+param set-default FW_PR_I 0.3
+param set-default FW_PR_P 0.08
+
+param set-default FW_RR_FF 0.05
+param set-default FW_RR_I 0.2
+param set-default FW_RR_P 0.03
+
+param set-default FW_YR_FF 0.3
+param set-default FW_YR_I 0.4
+param set-default FW_YR_P 0.2
 
 param set-default FW_PSP_OFF 2
 param set-default FW_P_LIM_MAX 32
 param set-default FW_P_LIM_MIN -15
 
-param set-default FW_RR_FF 0.5
-param set-default FW_RR_P 0.3
-param set-default FW_RR_I 0.5
-
-param set-default FW_YR_FF 0.5
-param set-default FW_YR_P 0.6
-param set-default FW_YR_I 0.5
-
 param set-default FW_SPOILERS_LND 0.4
 
 param set-default FW_THR_MIN 0.05
 param set-default FW_THR_TRIM 0.25
+param set-default FW_THR_MAX 0.6
 
-param set-default FW_T_CLMB_MAX 8
+param set-default FW_T_CLMB_R_SP 5
+param set-default FW_T_CLMB_MAX 6
 param set-default FW_T_SINK_MAX 2.7
 param set-default FW_T_SINK_MIN 2.2
 
 param set-default FW_W_EN 1
 
-param set-default MIS_TAKEOFF_ALT 30
-
-param set-default NAV_ACC_RAD 15
 param set-default NAV_DLL_ACT 2
 
 param set-default RWTO_TKOFF 1
 
 param set-default CA_AIRFRAME 1
 
 param set-default CA_ROTOR_COUNT 1
-param set-default CA_ROTOR0_PX 0.3
```

Comment:
Not that it matters, but why does this make a difference?

Reply:
Well the other way around, why should we keep a param in the config that doesn't have any effect?

Reply:
Interesting, I was coming more whether we "should" add PZ so that the pitching effects from throttle is simulated. Probably not relevant for this PR anyway.

Reply:
For that you'd though need to model/tune your CT setting. Sounds dangerous to me, and maybe would not even work yet. For differential thrust we need https://github.com/PX4/PX4-Autopilot/pull/23245 first.